src/docs/build-deploy-and-maintain-apps/build-with-rhel-base-images.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Build with RHEL Base Images
+
+slug: build-with-rhel-base-images
+
+description: Describes how to build images with RHEL as the base and leverage the Platforms subscriptions.
+
+keywords: images, RHEL, base images, managed images
+
+page_purpose: Describes how to build images with RHEL as the base and leverage the Platforms subscriptions.
+
+audience: developer
+
+author: Steven Barre
+
+content_owner: Matt Spencer
+
+sort_order: 10
+---
+
+# Build with RHEL Base Images
+
+If you wish to use a [Red Hat Enterprise Linux (RHEL)](https://catalog.redhat.com/software/containers/search?p=1&architecture=amd64&vendor_name=Red%20Hat%7CRed%20Hat%2C%20Inc.) based image to build upon, you will need to take some extra steps to get access to the RHEL Subscription included in the OpenShift Platform.
+
+Popular choices for RHEL base images are [ubi8](https://catalog.redhat.com/software/containers/ubi8/5c647760bed8bd28d0e38f9f) and [openshift4/ose-cli](https://catalog.redhat.com/software/containers/openshift4/ose-cli/5cd9ba3f5a13467289f4d51d).
+
+Reminder: you can pull these via [Artifactory](artifacts.developer.gov.bc.ca/redhat-docker-remote) as well via `artifacts.developer.gov.bc.ca/redhat-docker-remote`.

```

Comment:
Are these two Artifcatory links intended to be the same here? I get a 404 when clicking the hyperlink, is that expected? If so maybe we shouldn't hyperlink it and just show the address?

Reply:
Nope, for the first one I wanted to link to the Artifactory docs https://github.com/BCDevOps/developer-experience/blob/master/apps/artifactory/DEVHUB-README.md (or wherever they might be published)